setup.cfg

```diff
@@ -32,7 +32,7 @@ install_requires =
     pint
     pyvisa >= 1.8
     pyserial >= 2.7
-    pyqtgraph >= 0.9.10
+    pyqtgraph >= 0.12
```

Comment:
Please add a changelog note as I requested, so we don't overlook this.

Reply:
Ok, added changes to the CHANGES.rst. I added some other GUI changes I know have been merged recently.